src/cljs/witan/ui/controllers/datastore.cljs

```diff
@@ -127,7 +127,7 @@
   (if (:error data)
     (let [id (first (get-in data [:original :params]))
           tries (data/get-in-app-state :app/datastore :ds/query-tries)]
-      (if (< tries 3)
+      (if (< tries 5)
```

Comment:
This related to https://www.pivotaltracker.com/story/show/138820445 - it's a patch which hopefully means no more '401' visible issues